tools/tf2ngraph.py

```diff
@@ -173,9 +221,7 @@ def save_model(gdef, format, location):
     }[format](gdef, location)
 
 
-def attach_device_and_ng_backend(gdef, ng_backend):
-    ngraph_bridge.set_backend(ng_backend)
```

Comment:
removing `ngraph_bridge.set_backend`. The right way to set backend for grappler build is through rewriter config